src/bp/core/repositories/sessions.ts

```diff
@@ -107,7 +109,6 @@ export class KnexSessionRepository implements SessionRepository {
       .where('botId', botId)
       .andWhere(this.database.knex.date.isBefore('session_expiry', new Date()))
       .del()
-      .then()
   }
```

Comment:
We could just return it directly without awaiting, it's the calling function which should be awaiting

Reply:
it's fine with await because it makes it return a promise anyway without actually leaking the DB return result